openreview/venue_request/venue_request.py

```diff
@@ -1135,35 +1135,41 @@ def setup_request_form(self):
                 'default': ['Program chairs and paper authors only'],
                 'required': True
             },
+            'hide_fields': {
+                'values-regex': '.*',
+                'required': False,
+                'description': 'Comma separated values of submission fields to be hidden, author names are already hidden.',
```

Comment:
should we say that they will be hidden from the readers of the submission? and only visible to the PCs and authors?